synapse/lib/stormhttp.py

```diff
@@ -114,6 +114,12 @@ class LibHttp(s_stormtypes.Lib):
                        'default': None, },
                       {'name': 'body', 'type': 'bytes', 'desc': 'The data to post, as binary object.',
                        'default': None, },
+                      {'name': 'fields', 'type': 'list',
+                       'desc': 'A list of info dictionaries containing the name, value or sha256, '
+                               'and additional parameters for fields to post, as multipart/form-data.'
+                               'If a sha256 is specified, the corresponding file from the axon will'
```

Comment:
We need to note the **axon** is the component doing the HTTP activity, and not the Cortex. It is possible, for example, for different proxy or SSL CA settings to be deployed between those two components, so we need to be 100% clear how the fields entry behaves for users.

Reply:
updated desc to clarify